### PR TITLE
Landing pages navigation

### PR DESCRIPTION
**Pull request by SodhiA1**
I have added routing and the initial navigation pages. Terms.vue and status.vue are empty placeholders.

### CODE REVIEW

File: flapp-webapp/src/App.vue [hunk 1]
```diff
@@ -1,16 +1,21 @@
 <template>
   <div class="app-outer fill-body" id="app">
-    <FlappSurveys></FlappSurveys>
+    <NavigationTopbar />
```
Comment (andrewwhitehead) [inline flapp-webapp/src/App.vue:3]: FYI although the component is registered as NavigationTopbar, Vue converts that internally to navigation-topbar. This name should be preferred when referencing it from a template as names with hyphens will not conflict with native HTML elements.

File: flapp-webapp/src/routes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import LandingPage from './components/landingNavigation/LandingPage.vue';
+import login from './components/landingNavigation/login.vue';
```
Comment (andrewwhitehead) [inline flapp-webapp/src/routes.js:2]: Please name the component files consistently, with title case.

File: flapp-webapp/src/routes.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import serviceLocator from './components/landingNavigation/serviceLocator.vue';
+import FlappSurveys from './components/FlappSurveys.vue';
+import status from './components/status/status.vue'
+
+const routes = [
+    { path: "/", redirect: {name: "LandingPage"}},
+    { path: '/', name: 'LandingPage', component: LandingPage },
```
Comment (andrewwhitehead) [inline flapp-webapp/src/routes.js:9]: Duplicate root path?

File: flapp-webapp/src/routes.js (hunk 2 above)
Comment (SodhiA1) [reply flapp-webapp/src/routes.js:2]: Fixed in the next pull request

File: flapp-webapp/src/routes.js (hunk 3 above)
Comment (SodhiA1) [reply flapp-webapp/src/routes.js:9]: Fixed in the next pull request

File: flapp-webapp/src/App.vue (hunk 1 above)
Comment (SodhiA1) [reply flapp-webapp/src/App.vue:3]: @anissa-agahchen

**Pull request merged**